test/test_language.rb

```diff
@@ -426,6 +426,14 @@ def test_all_languages_have_a_language_id_set
     assert missing.empty?, message
   end
 
+  def test_all_languages_have_a_valid_id
+    invalid = Language.all.select { |language| language.language_id < 0 || language.language_id >= (2**31 - 1) }
+
+    message = "The following languages do not have a valid language_id. Please use script/set-language-ids --update as per the contribution guidelines.\n"
+    invalid.each { |language| message << "#{language.name}\n" }
+    assert invalid.empty?, message
+  end
+
```

Comment:
It'd probably also be worth testing that all the IDs are unique. A collision should be rare, but not impossible.